eng/Tools.props

```diff
@@ -49,6 +49,7 @@
     <!-- analyzers -->
     <PackageReference Include="Desktop.Analyzers" Version="1.1.0" />
     <PackageReference Include="Microsoft.CodeAnalysis.Common" Version="2.9.0" />
+    <PackageReference Include="Microsoft.DotNet.CodeAnalysis" Version="$(MicrosoftDotNetCodeAnalysisPackageVersion)" />
```

Comment:
What is the purpose of this change and why did you put it in the analyzers section 😕 

I'd expect all the dependencies of GenFacades to live next to the task itself.